pulp_rpm/tests/functional/api/test_sync.py

```diff
@@ -870,7 +872,8 @@ def test_complete_mirror_with_delta_metadata_fails(init_and_sync):
     Otherwise we would be mirroring the metadata without mirroring the DRPM packages.
     """
     with pytest.raises(PulpTaskError) as exc:
-        init_and_sync(url=DRPM_UNSIGNED_FIXTURE_URL, sync_policy="mirror_complete")
+        pass
+        # init_and_sync(url=DRPM_UNSIGNED_FIXTURE_URL, sync_policy="mirror_complete")
```

Comment:
can you remove this commented out line instead?
you anyway deleted this from constants